p2p/discover/v4_udp.go

```diff
@@ -63,15 +65,15 @@ const (
 
 // RPC packet types
 const (
-	pingPacket = iota + 1 // zero is 'reserved'
-	pongPacket
-	findnodePacket
-	neighborsPacket
+	p_pingV4 = iota + 1 // zero is 'reserved'
```

Comment:
I would add a `reservedV4Packdet` field here and start the `iota` there if it were me.